provisioning/modules/chromedriver/manifests/init.pp

```diff
@@ -0,0 +1,36 @@
+# install chromedriver for functional tests
+# we ship our local copy of chromedriver
+# because latest versions aren't working very well
+
+class chromedriver {
+
+  file { '/tmp/google-chrome-stable_54.0.2840.100-1_amd64.deb':
+    source => 'puppet:///modules/chromedriver/google-chrome-stable_54.0.2840.100-1_amd64.deb',
+    owner  => 'root',
+    group  => 'root',
+    mode   => '0755',
+  }
+
+  file { '/tmp/chromedriver_2.27_linux64.zip':
+    source => 'puppet:///modules/chromedriver/chromedriver_2.27_linux64.zip',
+    owner  => 'root',
+    group  => 'root',
+    mode   => '0755',
+  }
+
+  exec { 'install_google_chrome':
```

Comment:
Oh, you're right, I oversaw that one. I don't know if this is possible, so we can stick to an exec here.
To don't run the exec every time pls add sth like this:

```
    unless => '/usr/bin/dpkg -l chromedriver > /dev/null 2>&1'
```